website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Amazon Web Services (AWS)"
+linkTitle: "AWS"
+weight: 10
+---
+
+## Control Provisioning with Labels
+
+The [Provisioner CRD]({{< ref "provisioner-crd.md" >}}) supports defining
+instance types, node labels, and node taints.
+
+For certain well-known labels (documented below), Karpenter will provision
+nodes accordingly. For example, in response to a label of
+`topology.kubernetes.io/zone=us-east-1c`, Karpenter will provision nodes in
+that availability zone.
+
+Regarding AWS, 3 types of provisioning constraints are recognized: 
+- [Instance Type](#instance-type-allowlist)
+- [Node Labels](#node-lables-in-provisioner-spec) (e.g., Architecture, Capacity Type)
+- [Pod Labels](#pod-labels) (e.g., GPU)
+
+## Instance Type Allowlist
+
+Karpenter supports specifying [AWS instance type](https://aws.amazon.com/ec2/instance-types/). 
+
+If one instance type is listed, Karpenter will always provision that type.
+
+If more than one type is listed, Karpenter will intelligently select the
+instance type to maximize node utilization.
+
+The default value includes all instance types with the exclusion of metal
+(non-virtualized),
+[non-HVM](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html),
+and GPU instances. View the full list with `aws ec2 describe-instance-types`.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  instanceTypes:
+    - m5.large
+```
+
+*Override with pod.yaml*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.k8s.aws/instance-type: m5.large
+```
+
+## Node Lables in Provisioner Spec
+
+### Launch Template
+
+Karpenter uses [AWS Bottlerocket](https://aws.amazon.com/bottlerocket/) by default. 
+
+You can specify a different [launch
+template](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html),
+such as a customized version of Bottlerocket or a different platform
+altogether.
+
+**Template ID**
+- key: `node.k8s.aws/launch-template-id`
+- value example: `bottlerocket` (default)
+- value list: `aws ec2 describe-launch-templates`

```

Comment:
We should specify that Karpenter automatically creates a launch template with the name `Karpenter-<cluster name>-uuid` in the account/region where it is provisioned.